compiler/rustc_feature/src/accepted.rs

```diff
@@ -49,6 +49,8 @@ declare_features! (
     /// Allows the sysV64 ABI to be specified on all platforms
     /// instead of just the platforms on which it is the C ABI.
     (accepted, abi_sysv64, "1.24.0", Some(36167), None),
+    /// Allows using ADX intrinsics from `core::arch::{x86, x86_64}`.
+    (accepted, adx_target_feature, "1.60.0", Some(44839), None),
```

Comment:
This is the one thing I was most unclear on. It seems like the version number should match when the status last changed, in this case to `accepted`?